ui/components/multichain/import-nfts-modal/import-nfts-modal.js

```diff
@@ -77,13 +97,24 @@ export const ImportNftsModal = ({ onClose }) => {
   const [nftAddFailed, setNftAddFailed] = useState(false);
   const trackEvent = useContext(MetaMetricsContext);
 
+  const [actionMode, setActionMode] = useState(ACTION_MODES.IMPORT_NFT);
+
+  const [selectedNetworkForCustomImport, setSelectedNetworkForCustomImport] =
+    useState(null);
+
+  const allNetworks = useSelector(getNetworkConfigurationsByChainId);
+  const networkConfigurations = useSelector(getNetworkConfigurationsByChainId);
```

Comment:
these two variables are using the same selector

Reply:
https://github.com/MetaMask/metamask-extension/pull/32020/commits/08a6bbcbe39ff3879f13e85621cf7645b3cf8b1d